testing/e2e/conftest.py

```
@@ -1,88 +1,88 @@
+    
+    # In Docker mode, the mock service is already running via docker-compose
+    if is_docker_mode():
+        yield
+        return
+    
+    # In local mode, start the mock server serving from testdata
+    testdata_dir = Path(__file__).parent / "testdata"
+    if not testdata_dir.exists():
+        pytest.skip(f"testdata directory not found at {testdata_dir}")
+    
+    server = start_mock_server(testdata_dir)
```

Ruff (0.15.2)

[error] Local variable `server` is assigned to but never used

Remove assignment to unused variable `server`

(F841)